src/product/edit.tsx

```diff
@@ -34,14 +38,19 @@ function EditScreen() {
 
   return (
     <View>
-      <Form defaultValues={product} onSubmit={edit}>
+      <Form defaultValues={formProduct(product)} onSubmit={edit}>
+        <Label>Select an image</Label>
+        <ImgPicker name="image"/>
+        <Label>Product Name</Label>
         <Input name="name" placeholder="Name of the product" />
-        <Input name="price" type="decimal-pad" placeholder="Price" />
-        <Input name="stock" type="decimal-pad" placeholder="Amount in Stock" />
+        <Label>Price</Label>
+        <Input name="price" type="numeric" placeholder="Price" />
+        <Label>Stock</Label>
+        <Input name="stock" type="numeric" placeholder="Amount in Stock" />
+        <Label>Unit</Label>
         <Select name="unit" options={['kg', 'unite']} />
```

Comment:
We could extract this part in a component so that we can use it in both Create and Update, without rewriting. 
I can do it.

Reply:
Done